google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java

```diff
@@ -231,6 +230,34 @@ private LogEntry logEntryFor(ILoggingEvent e) {
     return builder.build();
   }
 
+  @InternalApi
```

Comment:
Add text to the annotation ("Visible for testing")

Reply:
Done